src/main/java/com/bittercode/service/impl/UserServiceImpl.java

```diff
@@ -37,10 +42,11 @@ public User login(UserRole role, String email, String password, HttpSession sess
             ps.setString(3, userType);
             ResultSet rs = ps.executeQuery();
             if (rs.next()) {
+                //Setter methods will never set a value as null (check implementation)
```

Comment:
I think they will, iff their input is null. So, the reason this is safe is that the inputs to the setter methods are nonnull.